package.json

```diff
@@ -62,8 +62,8 @@
     "parse5": "^2.2.3",
     "path-is-inside": "^1.0.2",
     "plylog": "^0.5.0",
-    "polymer-analyzer": "^2.7.0",
-    "polymer-bundler": "^3.1.1",
+    "polymer-analyzer": "^3.0.0-pre.17",
```

Comment:
Do we want `=` instead of `^`? Analyzer 3 is still getting breaking changes kinda often.

Reply:
Locked analyzer.  Left bundler as ^ tho, since I don't expect to be breaking its interface.